tracing-subscriber/src/registry/sharded.rs

```diff
@@ -116,7 +116,7 @@ fn id_to_idx(id: &Id) -> usize {
 /// 2. The `CloseGuard` is dropped at the end of `Subscriber::on_close`. On
 ///    drop, `CloseGuard` checks thread-local `CLOSE_COUNT`. If
 ///    `CLOSE_COUNT` is 0, the `CloseGuard` removes the span with the
-///    `id` from the registry, as all `Layers` that might have seen the
+///    `id` from the registry, as all `Subscribers` that might have seen the
```

Comment:
should probably be:
```suggestion
///    `id` from the registry, as all subscribers that might have seen the
```
and feel free to add a link, but it's not a huge deal here because this is an internal type...
an